sdk/tables/perf-tests/data-tables/test/createSimpleEntity.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { TablesTest } from "./tables.spec";
+import { TableEntity } from "@azure/data-tables";
+import { createBaseEntity } from "./utils/createBaseEntity";
+
+export class CreateSimpleEntityTest extends TablesTest {
+  public options: PerfStressOptionDictionary = {};
+
+  constructor() {
+    super("SimpleEntityPerf");
+  }
+
+  public async globalSetup() {
+    await super.globalSetup(); // Calling base class' setup
+  }
+
+  public async globalCleanup() {
+    await super.globalCleanup();
+  }
+
+  async runAsync(): Promise<void> {
+    const simpleEntity: TableEntity = createBaseEntity();
+    await this.client.createEntity(simpleEntity);
+  }
```

Comment:
Make runAsync as minimal as possible.
And to be able to mock the runAsync method, the request has to remain the same.
I recommend putting a static simpleEntity variable instead.

Reply:
Spoke offline with @HarshaNalluru, the Tables service doesn't allow receiving the same request twice as it requires every entity to have a unique combination of partitionKey and rowKey. While playback mode would work, running the tests live won't work